tests/modules/macros.rs

```diff
@@ -28,7 +28,7 @@ fn standard_format() {
         "code": Bson::JavaScriptCode("function(x) { return x._id; }".to_owned()),
         "i32": 12,
         "i64": -55,
-        "timestamp": Bson::TimeStamp(229_999_444),
+        "timestamp": Bson::TimeStamp(TimeStamp { time: 0, increment: 229_999_444 }),
```

Comment:
`229_999_444` is below `u32::MAX`, so this is the correct way to specify the old value